_tutorials/2016-05-10-iosxr-ansible.md

```diff
@@ -40,88 +52,79 @@ Vagrant.configure(2) do |config|
   config.vm.define "ubuntu" do |ubuntu|
     ubuntu.vm.box = "ubuntu/trusty64"
     ubuntu.vm.network :private_network, virtualbox__intnet: "link1", ip: "10.1.1.10"
+    ubuntu.vm.provision :shell, path: "ubuntu.sh", privileged: false
+    ssh_pub_key = File.readlines("#{Dir.home}/.ssh/id_rsa.pub").first.strip
```

Comment:
You've commented out the "echo #{ssh_pub_key >> ..." section later on. So this line is not required, right? Remove it.